src/utils/index.js

```diff
@@ -190,7 +190,7 @@ const getOptionsData = (options) => {
 };
 
 const formatCollaboratorOptions = (options) => (
-  options.map(({ name, uuid, image: { original: { url } } }) => ({ name, uuid, image_url: url }))
+  options.map(({ name, uuid, image: { original: { url = 'https://www.edx.org/images/edx-logo-header.png' } } }) => ({ name, uuid, image_url: url }))
```

Comment:
Hmm, thinking about this... is this file the right size or close to it?  Also, it might be a good idea to add a comment as to why this is necessary.  Or maybe there is more debugging we could add to help track this issue down?

Reply:
I can look closer and see how the logos are getting deleted. I'll timebox that effort to mid day tomorrow and see what I can find.